### PR TITLE
Update ldap.md: add a link to ldap.go

### PR DESCRIPTION
**Pull request by GreatBahram**
Hi there,

## Description
I updated this document because a lot of people, including myself, find it hard to follow this document.
I think a link to the `ldap.go`, which is just an example, will help readers to be able to test this feature more easily.

I think a link to the [minio-ldap-setting](https://github.com/donatello/minio-ldap-testing) can be helpful as well.

### CODE REVIEW

File: docs/sts/ldap.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ $ export MINIO_IDENTITY_LDAP_GROUP_SEARCH_FILTER='(&(objectclass=group)(member=%
 $ export MINIO_IDENTITY_LDAP_GROUP_NAME_ATTRIBUTE='cn'
 $ minio server ~/test
 ```
-
+You can make sure it works appropriately using our [Golang example program](ldap.go):
```
Comment (harshavardhana) [inline docs/sts/ldap.md]:
```suggestion
You can make sure it works appropriately using our [example program](https://raw.githubusercontent.com/minio/minio/master/docs/sts/ldap.go):
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#03a99993-3754-4d47-856e-6583ddadc024) |
| mint-zoned.sh | :x: [more...](#cfb3dd34-8e77-4829-8ebf-2c3b2044250f) |
<a name="03a99993-3754-4d47-856e-6583ddadc024"></a>
### 11409-35e8c70/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev8.minio.io:31330
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 8a093a7d0874:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 9 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 3 minutes and 42 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 minutes and 0 seconds
(5/15) Running awscli tests ... done in 4 minutes and 44 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 15 minutes and 30 seconds
(8/15) Running minio-dotnet tests ... done in 5 minutes and 28 seconds
(9/15) Running minio-go tests ... done in 28 minutes and 48 seconds
(10/15) Running minio-java tests ... FAILED in 9 minutes and 20 seconds
{
  "name": "minio-java",
  "function": "putObject()",
  "args": "[user metadata]",
  "duration": 171,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = AuthenticationFailed, message = -> github.com/Azure/azure-storage-blob-go/azblob.newStorageError, github.com/Azure/azure-storage-blob-go@v0.10.0/azblob/zc_storage_error.go:42\n===== RESPONSE ERROR (ServiceCode=AuthenticationFailed) =====\nDescription=Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.\nRequestId:c47c15a4-f01e-0022-5059-f983d8000000\nTime:2021-02-02T11:47:48.1960322Z, Details: \n   AuthenticationErrorDetail: The MAC signature found in the HTTP request 'OtHh3a+zeFu8THpOuBOuu3ifSkskPZD9azkr5enTUkQ=' is not the same as any computed signature. Server used following string to sign: 'PUT\n\n\n128\n\napplication/xml\n\n\n\n\n\n\nx-ms-blob-cache-control:\nx-ms-blob-content-disposition:\nx-ms-blob-content-encoding:\nx-ms-blob-content-language:\nx-ms-blob-content-type:application/octet-stream\nx-ms-client-request-id:060f1129-0e3f-4d78-4124-90263f9d8b70\nx-ms-date:Tue, 02 Feb 2021 11:47:48 GMT\nx-ms-meta-my_header1:a   b   c\nx-ms-meta-my_header2:\"a   b   c\"\nx-ms-meta-my_project:Project One\nx-ms-meta-my_unicode_tag:å•†å“\nx-ms-version:2019-02-02\n/minioazure/minio-java-test-3klarlo/minio-java-test-3ugtdjc\ncomp:blocklist\ntimeout:1501'.\n   Code: AuthenticationFailed\n   PUT https://minioazure.blob.core.windows.net/minio-java-test-3klarlo/minio-java-test-3ugtdjc?comp=blocklist&timeout=1501\n   Authorization: REDACTED\n   Content-Length: [128]\n   Content-Type: [application/xml]\n   User-Agent: [APN/1.0 MinIO/1.0 MinIO/2021-02-02T10:31:45Z]\n   X-Ms-Blob-Cache-Control: []\n   X-Ms-Blob-Content-Disposition: []\n   X-Ms-Blob-Content-Encoding: []\n   X-Ms-Blob-Content-Language: []\n   X-Ms-Blob-Content-Type: [application/octet-stream]\n   X-Ms-Client-Request-Id: [060f1129-0e3f-4d78-4124-90263f9d8b70]\n   X-Ms-Date: [Tue, 02 Feb 2021 11:47:48 GMT]\n   X-Ms-Meta-My_header1: [a   b   c]\n   X-Ms-Meta-My_header2: [\"a   b   c\"]\n   X-Ms-Meta-My_project: [Project One]\n   X-Ms-Meta-My_unicode_tag: [商品]\n   X-Ms-Version: [2019-02-02]\n   --------------------------------------------------------------------------------\n   RESPONSE Status: 403 Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.\n   Content-Length: [1092]\n   Content-Type: [application/xml]\n   Date: [Tue, 02 Feb 2021 11:47:48 GMT]\n   Server: [Microsoft-HTTPAPI/2.0]\n   X-Ms-Error-Code: [AuthenticationFailed]\n   X-Ms-Request-Id: [c47c15a4-f01e-0022-5059-f983d8000000]\n\n\n, bucketName = minio-java-test-3klarlo, objectName = minio-java-test-3ugtdjc, resource = /minio-java-test-3klarlo/minio-java-test-3ugtdjc, requestId = 165FEBCF81B86A41, hostId = 17b05153-8346-41ae-bcf7-388895c69217)\nrequest={method=PUT, url=http://minio-dev8.minio.io:31330/minio-java-test-3klarlo/minio-java-test-3ugtdjc, headers=x-amz-meta-My-Unicode-Tag: 商品\nx-amz-meta-My-Project: Project One\nx-amz-meta-My-header1: a   b   c\nx-amz-meta-My-Header2: \"a   b   c\"\nContent-Type: application/octet-stream\nHost: minio-dev8.minio.io:31330\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.0.3\nContent-MD5: A9oFTxee7YVcJ9fWsgQeKg==\nx-amz-content-sha256: 1ff7959f86334ddc5c188a5083268f600146328b2b6c5185e75bf7d9387d6b74\nx-amz-date: 20210202T114748Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20210202/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date;x-amz-meta-my-header1;x-amz-meta-my-header2;x-amz-meta-my-project;x-amz-meta-my-unicode-tag, Signature=*REDACTED*\n}\nresponse={code=403, headers=Accept-Ranges: bytes\nContent-Length: 3086\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nServer: MinIO\nVary: Origin\nX-Amz-Request-Id: 165FEBCF81B86A41\nX-Xss-Protection: 1; mode=block\nDate: Tue, 02 Feb 2021 11:47:48 GMT\n}\n >>> [io.minio.MinioClient.execute(MinioClient.java:775), io.minio.MinioClient.putObject(MinioClient.java:4547), io.minio.MinioClient.putObject(MinioClient.java:2713), io.minio.MinioClient.putObject(MinioClient.java:2830), FunctionalTest.testPutObject(FunctionalTest.java:763), FunctionalTest.putObject(FunctionalTest.java:890), FunctionalTest.runObjectTests(FunctionalTest.java:3751), FunctionalTest.runTests(FunctionalTest.java:3783), FunctionalTest.main(FunctionalTest.java:3927)]"
}
(10/15) Running minio-js tests ... FAILED in 1 minutes and 26 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 86,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:79:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:156:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:80:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (internal/streams/readable.js:684:10) at endReadableNT (internal/streams/readable.js:1327:12) at processTicksAndRejections (internal/process/task_queues.js:80:21)"
}
(10/15) Running minio-py tests ... done in 25 minutes and 51 seconds
(11/15) Running s3cmd tests ... done in 4 minutes and 59 seconds
(12/15) Running s3select tests ... done in 1 minutes and 25 seconds
(13/15) Running security tests ... done in 0 seconds

Executed 13 out of 15 tests successfully.
```
<a name="cfb3dd34-8e77-4829-8ebf-2c3b2044250f"></a>
### 11409-35e8c70/mint-zoned.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:32086
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fd2da581a487:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 44 seconds
(4/15) Running aws-sdk-ruby tests ... done in 4 seconds
(5/15) Running awscli tests ... done in 2 minutes and 20 seconds
(6/15) Running healthcheck tests ... done in 1 seconds
(7/15) Running mc tests ... done in 55 seconds
(8/15) Running minio-dotnet tests ... done in 46 seconds
(9/15) Running minio-go tests ... FAILED in 1 minutes and 25 seconds
{
  "args": {
    "bucketName": "minio-go-test-31ydwu0rx9htb2rt",
    "objectName": "gzc2djx02c9ziemqcjkagzmx5egchr",
    "objectPrefix": "",
    "recursive": "true"
  },
  "duration": 9845,
  "error": "Please reduce your request",
  "function": "RemoveObjects(bucketName, objectsCh, opts)",
  "message": "Cleanup failed",
  "name": "minio-go: testRemoveObjects",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 1 minutes and 25 seconds
(10/15) Running minio-js tests ... done in 1 minutes and 2 seconds
(11/15) Running minio-py tests ... done in 3 minutes and 8 seconds
(12/15) Running s3cmd tests ... done in 24 seconds
(13/15) Running s3select tests ... done in 8 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by eco-minio — APPROVED** [review]
LGTM
